pkgs/applications/editors/vim/plugins/overrides.nix

```
@@ -335,6 +335,10 @@ in
 
   blink-cmp-words = super.blink-cmp-words.overrideAttrs {
     dependencies = [ self.blink-cmp ];
+    meta = {
+      description = "A fast and offline word and synonym completion provider for Neovim";
```

```suggestion
      description = "Fast and offline word and synonym completion provider for Neovim";
```
I think technically, to follow contributing, we need to remove the `A` at the start.

---

Sure, happy to do that. I had copied it from the GitHub repo's description. Did not realize there is a contribution guideline around that.

---

Good to know. I've been trying to read through the various contributing guidelines, but you're right, there's a lot of it